lamindb_setup/_setup_user.py

```diff
@@ -40,50 +44,68 @@ def load_user(email: str | None = None, handle: str | None = None) -> str | None
 
     settings._user_settings = None  # this is to refresh a settings instance
 
-    return None
+    return user_settings
 
 
 def login(
-    user: str,
-    *,
-    key: str | None = None,
+    user: str | None = None, *, key: str | None = None, api_key: str | None = None
 ) -> None:
     """Log in user.
 
     Args:
         user: handle or email
         key: API key
+        api_key: Beta API key
     """
-    if "@" in user:
-        email, handle = user, None
+    if user is None and api_key is None:
+        if "LAMIN_API_KEY" in os.environ:
+            api_key = os.environ["LAMIN_API_KEY"]
```

Comment:
@falexwolf added the logic for `"LAMIN_API_KEY"`

Reply:
Great!

Reply:
Ok, merging this then.